### PR TITLE
add workflow and rake task to randomly lint a file

### PR DESCRIPTION
**Pull request by johrstrom**
Add workflow and rake task to randomly lint a file so that over time our files will not have so many linting errors. Current schedule is every 3 days.

┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1135148780858012/1203428702523468) by [Unito](https://www.unito.io)

### CODE REVIEW

File: lib/tasks/lint.rb [hunk 1]
```diff
@@ -7,18 +7,31 @@
...
+    "apps/**/*.rb",
+    "lib/**/*.rb",
+    "nginx_stage/**/*.rb",
+    "ood-portal-generator/**/*.rb",
+    "spec/**/*.rb",
+  ]
+
   begin
     require 'rubocop/rake_task'
     RuboCop::RakeTask.new(:rubocop, [:path]) do |t, args|
```
Comment (Oglopf) [inline lib/tasks/lint.rb:20]: I'm not sure where this object comes from. Do we use `rubocop-rake`? I don't see it in any `Gemfile` and I can't find any docs around this method of creating the object outside of what I found in the `rubocop-rake` repo's examples.

Comment (treydock) [reply lib/tasks/lint.rb:20]: https://www.rubydoc.info/gems/rubocop/RuboCop/RakeTask - this is a pretty common pattern for tools that provide custom Rake tasks to allow customizations of those Rake tasks.  We actually use this pattern in the OnDemand Packaging repo.

Comment (treydock) [reply lib/tasks/lint.rb:20]: I couldn't find this referenced on rubocop docs aside from references to how to enable rubocop plugins via Rake tasks.  Such as this: https://docs.rubocop.org/rubocop-rails/usage.html#rake-task

**Review by Oglopf — APPROVED** [review]
Cool stuff, looks good.

**Pull request merged**